client/tests/webdriver/baseSpecs/insights.spec.js

```diff
@@ -17,4 +17,36 @@ describe("Insights pages", () => {
         .true
     }
   })
+
+  it("Should open each insights page for super user", async() => {
+    await Insights.openAsSuperuser()
+    for (const insight of INSIGHTS) {
+      await (await Insights.getInsightsMenu()).waitForExist()
+      await (await Insights.getInsightsMenu()).waitForDisplayed()
+      await (await Insights.getInsightsMenu()).click()
+      await (await Insights.getInsightLink(insight)).click()
+      await (
+        await Insights.getAlert()
+      ).waitForDisplayed({ timeout: 1000, reverse: true })
+      // eslint-disable-next-line no-unused-expressions
+      expect(await (await Insights.getInsightDiv(insight)).isExisting()).to.be
+        .true
+    }
+  })
```

Comment:
Should call `await Insights.logout()` here if you want to continue the tests in the next section as another user.